docs/rules/no-hex-escape.md

```diff
@@ -0,0 +1,25 @@
+# Enforce the use of unicode encoded escapes instead of hexadecimal encoded escapes
+
+Enforces a convention of using unicode encoded escapes instead of hexadecimal encoded escapes to enfoce code consistency. Works on both Literals and TemplateLiterals.
```

Comment:
Drop the `Works on both Literals and TemplateLiterals.` part. It's already illustrated in the example.